docs/ossm/ambient/waypoint.md

```diff
@@ -0,0 +1,229 @@
+# Getting Started with Istio waypoint in Ambient Mode (Tech Preview)
+
+Once you have gotten started with Istio's ambient mode using ztunnel proxies, you may want to introduce waypoint proxies to take advantage of the many Layer 7 (L7) processing features that Istio provides. This section provides an overview and procedure for adding waypoint proxies to workloads running in your Istio ambient mode service mesh.
+
+---
+
+Ambient mesh splits Istio's functionality into two distinct layers, a secure overlay layer 4 (L4) and a Layer 7 (L7). The waypoint proxy is an optional component that is Envoy-based and handles L7 processing for workloads it manages. It acts as a gateway to a resource (a namespace, service or pod). Waypoint proxies are installed, upgraded and scaled independently from applications. They can be configured using the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/).
+
+## 1. What is a waypoint proxy for Istio's ambient mode?
+
+A **waypoint proxy** is an optional Envoy-based proxy that provides Layer 7 (L7) processing for specific workloads that are part of a service mesh running in ambient mode. Unlike the traditional sidecar model, where each workload gets its own Envoy proxy, waypoint proxies significantly reduce the number of proxies needed by allowing a single waypoint or set of waypoints to be shared across applications within a similar security boundary (e.g., all workloads in a namespace).
+
+Waypoint proxies operate independently of applications, meaning application owners don't need to be aware of their existence. In **ambient mode**, policies are enforced by the destination waypoint, which acts as a gateway, ensuring all incoming traffic to a resource (namespace, service, or pod) passes through it for policy enforcement.
+
+### 1.1 Do You Need a Waypoint Proxy?
+
+The **ztunnel node proxy** handles most of the features in ambient mode, primarily focusing on Layer 4 (L4) traffic processing. However, if your applications require any of the following L7 mesh functionalities, you'll need to use a waypoint proxy:
+
+* **Traffic Management:** This includes advanced HTTP routing and load balancing, circuit breaking, rate limiting, fault injection, retries, and timeouts.
+* **Security:** For rich authorization policies based on L7 primitives like request type or HTTP headers.
+* **Observability:** To collect HTTP metrics, enable access logging, and implement tracing for your applications.
+
+**Trade-offs:**
```

Comment:
I think we can remove the "Trade-offs" section here, as it is generic to ambient mode (not specific to waypoints) and is duplicated in the ZTunnel installation README.

Reply:
Removed.